### PR TITLE
chore: update colors and clean up some code

### PR DESCRIPTION
**Pull request by renaudmathieu**
🎨 New color scheme suggestion
🔧 Code cleanup

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
-    <state>
-        <option name="USE_PER_PROJECT_SETTINGS" value="true" />
-    </state>
+  <state>
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+  </state>
```
Comment (martinbonnin) [inline .idea/codeStyles/codeStyleConfig.xml:4]: Can we have a codestyle for the codestyle please 🤣 ?

JK, it's great 👍

**Review by martinbonnin — APPROVED** [review]
👍

**Comment by enthuan** [comment]
😍

**Comment by enthuan** [comment]
I was working on the font. I think Droidcon uses Montserrat. I will push it tomorrow

**Review by BoD — APPROVED** [review]

**Review by enthuan — APPROVED** [review]

**Pull request merged**